framework/components/ecl-carousels/test/spec/carousel.js

```diff
@@ -0,0 +1,58 @@
+const componentUrl = 'ecl-carousels.html';
```

Comment:
True! I don't know how to solve this... I increased the timeout to 4 seconds and they still won't appear except 1 browser which I least expect to be working ... The only idea I have is to have the images in the static assets folder to that they are accessible offline, but it I think that means they will go in the build and dist. I'm open for ideas here.